addons/website/models/website.py

```diff
@@ -851,7 +851,7 @@ def _handle_views_and_pages(views):
                 dependencies.setdefault(model_name, [])
                 dependencies[model_name] += [{
                     'field_name': Model.fields_get()[column]['string'],
-                    'record_name': rec.name,
+                    'record_name': getattr(rec, 'name', f'{model_name} {rec.id}'),
```

Comment:
Why not `display_name`? or
```suggestion
                    'record_name': rec.name if 'name' in rec else f'{model_name} {rec.id}',
```

Reply:
We changed for that, thank you